docs/manuals/source/manually_added_config_directive_descriptions/dir-job-Name.rst.inc

```diff
@@ -1,3 +1,7 @@
 The Job name. This name can be specified on the Run command in the console program to start a job. If the name contains spaces, it must be specified between quotes. It is generally a good idea to give your job the same name as the Client that it will backup. This permits easy identification of jobs.
 
-When the job actually runs, the unique Job Name will consist of the name you specify here followed by the date and time the job was scheduled for execution. This directive is required.
+When the job actually runs, the unique Job Name will consist of the name you specify here followed by the date and time the job was scheduled for execution.
+
+It is recommended to limit job names to 98 characters. Higher is always possible, but when the job is run, its name will be truncated to accomodate certain protocol limitations, as well as the above mentioned date and time.
+
+This directive is required.
```

Comment:
```suggestion
```
That the directive is required will be written automatically to the documentation.